be/src/io/fs/hdfs_file_system.cpp

```diff
@@ -61,6 +61,11 @@ class HdfsFileSystemCache {
     void _clean_oldest();
 };
 
+std::shared_ptr<HdfsFileSystem> HdfsFileSystem::create(const THdfsParams& hdfs_params,
+                                                       const std::string& path) {
```

Comment:
It is better to change the interface like this.

```suggestion
std::unique_ptr<HdfsFileSystem> HdfsFileSystem::create(const THdfsParams& hdfs_params,
                                                       const std::string& path) {
```